imports/plugins/core/revisions/server/hooks.js

```diff
@@ -537,6 +552,10 @@ Products.before.update(function (userId, product, fieldNames, modifier, options)
   }
 
   Revisions.update(revisionSelector, revisionModifier);
+  Hooks.Events.run("afterRevisionsUpdate", userId, {
+    ...productRevision,
+    workflow: { ...productRevision.workflow, status: "revisions/update" }
+  });
```

Comment:
This one may have a bunch of different changes in `revisionModifier` after going through the above, so unfortunately probably the best choice here is to `findOne` after the update.